### PR TITLE
[4.0] Translating media type

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30230

### Summary of Changes
This let's translate the `dir` and `file` types

### Testing Instructions
Set Debug Language on.
Load Media Manager, select a folder and click on the **`i`** to get the informations
Do the same for a file.

Patch. Run npm

### Actual result BEFORE applying this Pull Request
The `Type` is  not translated

Screen Shot 2020-07-31 at 11 21 14
Screen Shot 2020-07-31 at 11 21 03

### Expected result AFTER applying this Pull Request
Screen Shot 2020-07-31 at 11 18 32
Screen Shot 2020-07-31 at 11 18 16

### CODE REVIEW

File: administrator/components/com_media/tmpl/media/default_texts.php [hunk 1]
```diff
@@ -61,6 +61,8 @@
 Text::script('COM_MEDIA_TOGGLE_SELECT_ITEM', true);
 Text::script('COM_MEDIA_TOOLBAR_LABEL', true);
 Text::script('COM_MEDIA_UPLOAD_SUCCESS', true);
+Text::script('COM_MEDIA_FILE', true);
```
Comment (Quy) [inline administrator/components/com_media/tmpl/media/default_texts.php]: Please alpha sort.

File: administrator/components/com_media/resources/scripts/components/infobar/infobar.vue [hunk 2]
```diff
@@ -12,7 +12,12 @@
                 <dd>{{ item.directory }}</dd>
 
                 <dt>{{ translate('COM_MEDIA_MEDIA_TYPE') }}</dt>
-                <dd>{{ item.type || '-' }}</dd>
+                <dd v-if="item.type === 'file'">
+                  {{ translate('COM_MEDIA_FILE') }}</dd>
+                <dd v-else-if="item.type === 'dir'">
+                  {{ translate('COM_MEDIA_FOLDER') }}</dd>
+                <dd v-else>
```
Comment (Quy) [inline administrator/components/com_media/resources/scripts/components/infobar/infobar.vue]: Match line 30.

Comment (infograf768) [reply administrator/components/com_media/resources/scripts/components/infobar/infobar.vue]: Done and also simplified dd

**Comment by infograf768** [comment]
@Quy 
Done.

**Comment by Quy** [comment]
I am not on a computer to test. Can you check line 8 for translation?

**Comment by infograf768** [comment]
> Can you check line 8 for translation?

It is not even displayed here in source.

**Comment by Quy** [comment]
https://github.com/joomla/joomla-cms/blob/10e0737b4baabcefc10103cb18528da0be079dc0/administrator/components/com_media/resources/scripts/components/infobar/infobar.vue#L8

**Comment by infograf768** [comment]
I know where it is in the file. As I said, when the page is loaded, I do not see it displayed in source.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 10e0737b4baabcefc10103cb18528da0be079dc0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30243">issues.joomla.org/tracker/joomla-cms/30243</a>.</sub>

**Comment by Quy** [comment]
Ignore it for now. I will open an issue if I can reproduce it. Thanks.

**Comment by yasin-96** [comment]
I have tested this item :white_check_mark: successfully on 10e0737b4baabcefc10103cb18528da0be079dc0<br /><br />After Installed the patch and run the npm ci command the media type where translated correctly.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30243">issues.joomla.org/tracker/joomla-cms/30243</a>.</sub>

**Comment by david0296** [comment]
I have tested this item :white_check_mark: successfully on 10e0737b4baabcefc10103cb18528da0be079dc0<br /><br />After applying and running `npm ci`, the ***Type*** is translated<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30243">issues.joomla.org/tracker/joomla-cms/30243</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30243">issues.joomla.org/tracker/joomla-cms/30243</a>.</sub>

**Pull request merged**